### PR TITLE
Update weekly workflow to run perf tests

### PR DESCRIPTION
**Pull request by yunchu**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: .github/workflows/weekly.yml [hunk 1]
```diff
@@ -10,41 +10,23 @@ on:
...
-      tests-dir: ${{ matrix.test_dir }}
-      runs-on: "['self-hosted', 'Linux', 'X64', 'dmount']"
-      task: ${{ matrix.task }}
-      timeout-minutes: 8640
-      upload-artifact: true
-      artifact-prefix: "weekly-test-results"
+      model-type: default
+      data-size: small
+      num-repeat: 1
+      num-epoch: 3
```
Comment (goodsong81) [inline .github/workflows/weekly.yml]: medium / 1 / 3 should be default for speed benchmark. Is this for just sanity check?

File: .github/workflows/weekly.yml [hunk 2]
```diff
@@ -10,41 +10,23 @@ on:
...
+      eval-upto: optimize
+      artifact-prefix: weekly-perf-speed-benchmark
+  Performance-Accuracy-Tests:
+    name: Performance-Accuracy-py310
+    uses: ./.github/workflows/perf-accuracy.yml
+    with:
+      model-type: default
+      data-size: small
+      num-repeat: 0
+      num-epoch: 0
```
Comment (goodsong81) [inline .github/workflows/weekly.yml]: Similar here. (all / 0 / 0)

File: .github/workflows/weekly.yml (hunk 1 above)
Comment (yunchu) [reply .github/workflows/weekly.yml]: I want to see that the workflow is working as expected using the small set (to reduce time) then set them with required configuration later.

**Review by goodsong81 — DISMISSED** [review]

**Review by eunwoosh — DISMISSED** [review]

**Comment by eunwoosh** [comment]
LGTM. I have one question. I think speed test needs to use whole machines alone, but it seems not to be ensured currently. Do you have a plan to do that?

**Comment by yunchu** [comment]
> LGTM. I have one question. I think speed test needs to use whole machines alone, but it seems not to be ensured currently. Do you have a plan to do that?

currently only a machine runs as a ci-runner so only one workflow would be ran at any moment. only concern to the speed tests are the h/w differences (cpu, i9-11900 vs i9-12900k) across the all ci-runners.

**Comment by eunwoosh** [comment]
> > LGTM. I have one question. I think speed test needs to use whole machines alone, but it seems not to be ensured currently. Do you have a plan to do that?
> 
> currently only a machine runs as a ci-runner so only one workflow would be ran at any moment. only concern to the speed tests are the h/w differences (cpu, i9-11900 vs i9-12900k) across the all ci-runners.

That seems good, I think. we may be able to consider labelling single machine as speed test if necessary.

**Review by eunwoosh — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**